.github/workflows/templates/test-matrix.yml

```
@@ -12,8 +12,8 @@ permissions:
 
 jobs:
   prepare:
-    name: Prepare Poetry & Assured OSS
-    uses: ./.github/workflows/templates/prepare-poetry.yml
+    name: Prepare uv & Assured OSS
+    uses: ./.github/workflows/prepare-uv.yml
```

actionlint (1.7.12)

[error] error while parsing reusable workflow "./.github/workflows/prepare-uv.yml": "on:" is not found

(workflow-call)